app/models/graders/junit_grader.rb

```diff
@@ -50,6 +61,58 @@ def import_data_schema
     "junit_import_schema"
   end
 
+  def grade(assignment, submission, prio = 0)
+    begin
+      secret, secret_file_path = generate_orca_secret!(submission)
+      send_job_to_orca(submission, secret)
+    rescue OrcaJobCreationError => exception
+      FileUtils.rm(secret_file_path)
+      Audit.log("Failed to send job to orca on submission #{submission.id} for assignment #{assignment.id}.")
```

Comment:
Here my thought process was that we don't actually want to stop normal grading procedures if the request to Orca fails, but we should have some record of why it did not get sent over properly.